### PR TITLE
cudadeprecation 14&15

### PR DESCRIPTION
**Pull request by caryr35**
CUDA 14 and 15 Deprecation notices

### CODE REVIEW

**Comment by exactlyallan** [comment]
@caryr35 can this be closed?

File: _notices/rsn0027.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.02+"
+notice_created: 2023-01-27
```
Comment (aravenel) [inline _notices/rsn0027.md:22]:
```suggestion
notice_created: 2023-03-08
```

File: _notices/rsn0027.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-01-27
```
Comment (aravenel) [inline _notices/rsn0027.md:24]:
```suggestion
notice_updated: 2023-03-08
```

File: _notices/rsn0027.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+## Overview
+
+RAPIDS is deprecating `CUDA 11.4` and `CUDA 11.5` containers in its upcoming `v23.02` release. RAPIDS will plan to EOL these containers in a future release (currently targeting `v23.04`), at which point `CUDA 11.4` and `CUDA 11.5` containers will no longer be published.
```
Comment (aravenel) [inline _notices/rsn0027.md:29]:
```suggestion
RAPIDS is deprecating `CUDA 11.4` and `CUDA 11.5` containers effective its `v23.04` release. RAPIDS will plan to EOL these containers in a future release (currently targeting `v23.06`), at which point `CUDA 11.4` and `CUDA 11.5` containers will no longer be published.
```

File: _notices/rsn0027.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+
+## Impact
+
+Once these containers reach EOL (currently targeting `v23.04` release), RAPIDS will no longer publish these containers. Conda packages will still support `CUDA 11.4` and `CUDA 11.5` due to CUDA Enhanced Compatibility.
```
Comment (aravenel) [inline _notices/rsn0027.md:34]:
```suggestion
Once these containers reach EOL (currently targeting `v23.06` release), RAPIDS will no longer publish these containers. Note that conda packages will still support `CUDA 11.4` and `CUDA 11.5` due to CUDA Enhanced Compatibility.
```

File: _notices/rsn0027.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+notice_status: In Progess
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.02+"
```
Comment (aravenel) [inline _notices/rsn0027.md:21]:
```suggestion
notice_rapids_version: "v23.04+"
```

**Review by aravenel — CHANGES_REQUESTED** [review]

**Pull request merged**